src/SFA.DAS.ApplyService.Application.UnitTests/Handlers/UpdateGatewayReviewStatusAndCommentCommandHandlerTests/UpdateGatewayReviewStatusAndCommentCommandHandlerTests.cs

```diff
@@ -75,5 +80,21 @@ public async Task UpdateGatewayReviewStatusAndComment_when_rejected_creates_over
 
             _oversightReviewRepository.Verify(x => x.Add(It.Is<OversightReview>(r => r.ApplicationId == _applicationId && r.Status == OversightReviewStatus.Rejected)));
         }
+
+        [Test]
+        public async Task Handler_sends_updated_email()
```

Comment:
Is it worth adding another test (or set of TestCases) to verify that in cases other than Reject, _no_ email is sent?